### ISSUE TITLE
Site domain heya.fund is down

### ISSUE THREAD

**Issue by TheMhv** [verbiricha/goalz#2]
Apparently the `heya.fund` website domain has gone down

**Comment by verbiricha** [comment]
Yeah sorry about that, don't have the time/funds to support the project at the moment. I hope someone takes the code and deploys to an alternative domain.
